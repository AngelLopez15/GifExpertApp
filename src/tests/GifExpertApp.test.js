import React from 'react';
import '@testing-library/jest-dom'

const { shallow } = require("enzyme")
const { GifExpertApp } = require("../GifExpertApp")

describe('Pruebas en <GifExpertApp />', () => {
    test('debe de mostarse correctamente', () => {
        const wrapper = shallow(<GifExpertApp />)
        expect(wrapper).toMatchSnapshot()
    })
    
})
