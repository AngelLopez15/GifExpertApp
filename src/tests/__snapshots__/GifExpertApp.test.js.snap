// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp /> debe de mostarse correctamente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="one punch"
      key="one punch"
    />
  </ol>
</Fragment>
`;
